src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIO.java

```diff
@@ -48,7 +52,4 @@ public default void setLeftVelocitySetpoint(double velocityRadiansPerSecond) {}
 
```

Comment:
It is probably worth making this function setLeftGoalVelocity, as in a motion profile, the setpoint is determined by the profile constraints.